input/community/speakers/alyssa-nicoll.md

```diff
@@ -8,7 +8,6 @@ Topics:
   - Diversity & Inclusion
   - JavaScript
 Blog: https://www.telerik.com/blogs/author/alyssa-nicoll
-Feed: https://feeds.feedwrench.com/all-shows-devchattv.rss
```

Comment:
This feed does not appear to be valid RSS (it won't pass any of the validators) and it's 25 MB (!) as well. We could reach out and see if she's got another one, but for now I removed it since it was generating a lot of warnings.